setup.py

```diff
@@ -8,7 +8,7 @@
           "numpy>=1.22.3", "pytest", "gym==0.21.0", "matplotlib", "imageio",
           "imageio-ffmpeg", "pandas", "torch", "scipy", "tabulate", "dill",
           "pyperplan", "pathos", "requests", "slack_bolt", "pybullet>=3.2.0",
-          "sklearn", "pyqt5", "graphlib-backport", "openai", "pyyaml",
+          "sklearn", "graphlib-backport", "openai", "pyyaml",
```

Comment:
pyqt5 is only needed for collecting human demonstrations. this isn't actively being used. it's a flaming disaster to install on macs with m1 chips. so i'm just removing it for now